CMakeLists.txt

```diff
@@ -59,7 +59,26 @@ else()
 endif()
 
 target_include_directories(gfx PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
-target_link_libraries(gfx PRIVATE d3d12 dxgi)
+# Find the latest windows SDK needed for d3d12 libs
+if(NOT CMAKE_WINDOWS_KITS_10_DIR)
+    set(CMAKE_WINDOWS_KITS_10_DIR "C:/Program Files (x86)/Windows Kits/10")
+endif()
+if(CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION)
```

Comment:
When building for a visual studio target, cmake defines this to the targetted kit version so we reuse to match and thus has no change in behaviour for VS builds.